### PR TITLE
Include elapsedMillis when logging slow kube client requests.

### PR DESCRIPTION
**Pull request by dgoodwin**
Forgot to include the actual request time when logging the slow requests.

/assign @suhanime

### CODE REVIEW

File: pkg/controller/utils/clientwrapper.go [hunk 1]
```diff
@@ -119,11 +119,12 @@ func (cmt *ControllerMetricsTripper) RoundTrip(req *http.Request) (*http.Respons
...
-				"method":     req.Method,
-				"path":       path,
-				"remote":     remoteStr,
-				"status":     resp.Status,
+				"controller":    cmt.Controller.String(),
+				"method":        req.Method,
+				"path":          path,
+				"remote":        remoteStr,
+				"status":        resp.Status,
+				"elapsedMillis": applyTime * 1000, // millis for consistency with how we log controller reconcile time
```
Comment (staebler) [inline pkg/controller/utils/clientwrapper.go:127]: Change `applyTime` to be a `time.Duration`, and use `applyTime.Milliseconds()` here. That way the user does not have to figure out that `applyTime` is in seconds.

Comment (dgoodwin) [reply pkg/controller/utils/clientwrapper.go]: done

File: pkg/controller/utils/clientwrapper.go [hunk 2]
```diff
@@ -113,17 +113,18 @@ func (cmt *ControllerMetricsTripper) RoundTrip(req *http.Request) (*http.Respons
 	path, pathErr := parsePath(req.URL.Path)
 	// Call the nested RoundTripper.
 	resp, err := cmt.RoundTripper.RoundTrip(req)
-	applyTime := metav1.Now().Sub(startTime).Seconds()
+	applyTime := metav1.Now().Sub(startTime)
 	if err == nil && pathErr == nil {
 		metricKubeClientRequests.WithLabelValues(cmt.Controller.String(), req.Method, path, remoteStr, resp.Status).Inc()
-		metricKubeClientRequestSeconds.WithLabelValues(cmt.Controller.String(), req.Method, path, remoteStr, resp.Status).Observe(applyTime)
+		metricKubeClientRequestSeconds.WithLabelValues(cmt.Controller.String(), req.Method, path, remoteStr, resp.Status).Observe(applyTime.Seconds())
 		if applyTime >= 5.0 {
```
Comment (staebler) [inline pkg/controller/utils/clientwrapper.go:120]:
```suggestion
		if applyTime >= 5 * time.Second {
```

Comment (dgoodwin) [reply pkg/controller/utils/clientwrapper.go]: updated

**Review by staebler — APPROVED** [review]
/lgtm

**Comment by suhanime** [comment]
Looks like the bot skipped retest

/retest

**Pull request merged**